data/json/monstergroups.json

```diff
@@ -498,7 +498,7 @@
       { "monster" : "mon_zombie_hazmat", "freq" : 10, "cost_multiplier" : 3 },
       { "monster" : "mon_zombie_fireman", "freq" : 10, "cost_multiplier" : 2 },
       { "monster" : "mon_zombie_grabber", "freq" : 30, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_hunter", "freq" : 20, "cost_multiplier" : 5 },
+      { "monster" : "mon_zombie_hunter", "freq" : 10, "cost_multiplier" : 5 },
```

Comment:
Hunter and runner have same cost multiplier. Runner should probably be lower or spawn in packs or something like that.